src/subcommand/wallet/send.rs

```diff
@@ -66,6 +66,14 @@ impl Send {
         self.fee_rate,
         false,
       )?,
+      Outgoing::Sat(sat) => Self::create_unsigned_send_sat_transaction(
```

Comment:
You could probably just use the send_satpoint function and call wallet.find_sat_in_outputs(sat)? directly in the function arguments.

Reply:
nice tips